src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -16,15 +18,15 @@ public class FormulaValueJSON
         /// <summary>
         /// Convenience method to create a value from a json representation. 
         /// </summary>
-        public static FormulaValue FromJson(string jsonString)
+        public static FormulaValue FromJson(string jsonString, FormulaType formulaType = null)
```

Comment:
>formulaType [](http://example.com/codeflow?start=75&length=11)

IF this is non-null, does that mean that 

FromJson(json, type).Type == type?

Reply:
Some returned values are optional so this isn't true